custom-elements-manifest.config.js

```diff
@@ -39,8 +42,8 @@ export default {
               });
             });
 
-            const parsed = parse(customComments + '\n */');
-            parsed[0].tags?.map(t => {
```

Comment:
I think even theoretically `forEach` should have a performance benefit because it doesn't need to construct a new array. From some microbenchmarks online it looks like `forEach` is twice as fast as `map`. I'd be very confident to say that `forEach` will always be as fast if not faster than `map` given that it is doing less. Similar to how a normal for loop would be faster than `forEach` because it's even simpler since it doesn't need to create and call a function.

* [Array iterators microbenchmark](https://leanylabs.com/blog/js-forEach-map-reduce-vs-for-for_of/)
* [ECMAScript Spec for .map](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.map)
* [ECMAScript Spec for .forEach](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.foreach)

Reply:
Thanks. I'm happy to change my habits to use `forEach` when I don't need to return an array.